src/app/models/form.server.model.ts

```diff
@@ -163,10 +167,13 @@ const compileFormModel = (db: Mongoose): IFormModel => {
     {
       title: {
         type: String,
-        validate: [
-          /^[a-zA-Z0-9_\-./() &`;'"]*$/,
-          'Form name cannot contain special characters',
-        ],
+        validate: {
+          // validation fails if regex pattern exists in the title
+          validator: function (v: string) {
+            return !UNICODE_ESCAPED_REGEX.test(v)
+          },
```

Comment:
/super nit: use shorter syntax for a simple function like this
```suggestion
          validator: v => !UNICODE_ESCAPED_REGEX.test(v),
```

Reply:
More importantly: are you sure this test is correct here? `UNICODE_ESCAPED_REGEX` detects the incorrect utf8 characters by inspecting the json-encoded version of the value, but `v` here is the raw string value, no?